config/src/main/java/com/alibaba/nacos/config/server/enums/OperationType.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.config.server.enums;
+
+/**
+ * Operation type enum.
+ *
+ * @author dirtybit
+ */
+public enum OperationType {
+
+    /**
+     * Insert
+     */
+    INSERT("I"),
+
+    /**
+     * Update
+     */
+    UPDATE("U"),
+
+    /**
+     * Delete
+     */
+    DELETE("D"),
+
+    /**
+     * Select
+     */
+    SELECT("S");
```

Comment:
does SELECT make any sense？

Reply:
if a enum for opt ype created, replace all hard-code of "D","I","U"

Reply:
OK，I will do it.